LabApiUtilities/src/test/com/microsoft/identity/labapi/utilities/client/LabGuestAccountTest.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.identity.labapi.utilities.client;
+
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment;
+import com.microsoft.identity.labapi.utilities.constants.FederationProvider;
+import com.microsoft.identity.labapi.utilities.constants.GuestHomedIn;
+import com.microsoft.identity.labapi.utilities.constants.UserType;
+import com.microsoft.identity.labapi.utilities.exception.LabApiException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class LabGuestAccountTest {
+
+    @Test
+    public void testCanCreateLabGuestAccount() throws LabApiException {
+        final LabQuery queryForUserB = LabQuery.builder()
```

Comment:
seems like the only user in this test. We can remove the `B` from the variable name